integration-tests/common/test_common.go

```diff
@@ -4,42 +4,52 @@ import (
 	"context"
 	"encoding/hex"
 	"flag"
+	"fmt"
+	"github.com/smartcontractkit/chainlink-relay/pkg/logger"
+	"github.com/smartcontractkit/chainlink-starknet/relayer/pkg/chainlink/ocr2"
+	"github.com/smartcontractkit/chainlink-starknet/relayer/pkg/starknet"
+	"os"
 	"strings"
+	"time"
 
-	"github.com/dontpanicdao/caigo/gateway"
 	"github.com/go-resty/resty/v2"
 	. "github.com/onsi/gomega"
+	"github.com/rs/zerolog/log"
 	"github.com/smartcontractkit/chainlink-env/environment"
-	"github.com/smartcontractkit/chainlink-env/pkg/helm/chainlink"
-	"github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver"
-	mockservercfg "github.com/smartcontractkit/chainlink-env/pkg/helm/mockserver-cfg"
 	"github.com/smartcontractkit/chainlink-starknet/ops"
 	"github.com/smartcontractkit/chainlink-starknet/ops/devnet"
+	"github.com/smartcontractkit/chainlink-starknet/ops/gauntlet"
 	"github.com/smartcontractkit/chainlink-starknet/relayer/pkg/chainlink/keys"
 	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
 	ctfClient "github.com/smartcontractkit/chainlink-testing-framework/client"
 	"github.com/smartcontractkit/chainlink/integration-tests/client"
 )
 
-const (
-	ChainName = "starknet"
-	ChainId   = gateway.GOERLI_ID // default chainID for local devnet and live testnet
-)
-
 var (
-	err       error
-	clImage   string
-	clVersion string
+	err error
+	// Default version to run on
+	clImage   = "public.ecr.aws/chainlink/chainlink"
+	clVersion = "1.9.0"
```

Comment:
just curious, why are we setting defaults here? it feels like an easy mistake to forget to set it when running locally and then it it will use the defaults

maybe it's just better to throw an error if the image/version is not passed via ENV vars or CLI flags?

Reply:
The custom images we build have a TTL of around 1 week and we cant set them since tests will start failing so I just set this to the latest release candidate. Maybe just forcing an image version is the best option, or develop